intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -195,39 +200,131 @@ std::vector<int> Model::generate_tokens(const std::vector<int>& input_ids) {
   return output_ids;
 }
 
-int Model::post_process(float* logits) {
-  int alpha_frequency = 0;
-  int alpha_presence = 0;
+int Model::post_greedy_search(float* logits) {
+  int id = std::max_element(logits, logits + n_vocab) - logits;
+  return id;
+}
+
+int Model::post_beam_search(float* logits) {
+  // TODO: to implement
+  fprintf(stderr, "beam search is not supported!\n");
```

Comment:
due to `kv cache update func ptr` and `batch inference` implementations in different models. We may need to add a supported table here.

Reply:
update it when you do the beam search